app/views/homepage/index.haml

```diff
@@ -135,6 +135,10 @@
       - else
         .home-no-listings
           - if params[:q] || params[:category] || params[:share_type] # Some filter in use
-            = t(".no_listings_with_your_search_criteria")
+            = t(".no_listings_with_your_search_criteria",
```

Comment:
As I mentioned already in Flowdock, I feel that "Post a new listing" is not the right call to action, if no listings is found. Maybe it should be something that points user to loosen the search criteria?